README.md

```diff
@@ -1 +1,251 @@
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [MY PORTFOLIO] <a name="about-project"></a>
+
+My name is Esmael Mohammed, this project is my portfolio webpage project
+
+**[MY PORTFOLIO]** A responsive website called MY PORTFOLIO accurately describes who I am, my professional accomplishments, and my job history.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+>I used the following tech stacks to build this Portfolio
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="#">html</a></li>
+    <li><a href="#">css</a></li>
+  </ul>
+</details>
+ 
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+>I have 2 key feature called About, Projects
+
+- **[About]**
+- **[Projects]**
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+
+
+- [Live Demo Link](https://esmiz-dream.github.io/Portfolio/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+
+
+>To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+to setup git, github and code editor.
+
+<!--
+Example command:
+
+```sh
+ gem install rails
+```
+ -->
+
+### Setup
+
+>Clone this repository to your desired folder:
+
+<!--
+Example commands:
+
+```sh
+  cd my-folder
+  git clone git@github.com:myaccount/my-project.git
+```
+--->
+
+### Install
+
+Install this project with:
+git and github
+<!--
+Example command:
+
+```sh
+  cd my-project
+  gem install
+```
+--->
+
+### Usage
+
+To run the project, open your live server and open the link the live server gives to you.
+
+<!--
+Example command:
+
+```sh
+  rails server
+```
+--->
+
+ 
+<!--
+Example command:
+
+```sh
+  bin/rails test test/models/article_test.rb
+```
+--->
+
+### Deployment
+
+You can deploy this project using:
+any types of web server,
+
+<!--
+Example:
+
+```sh
+
+```
+ -->
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+ 
+👤 **Author1**
+
+- GitHub: [@githubhandle](https://github.com/esmiz-dream)
+- Linkedin: [@githubhandle](linkedin.com/in/esmael-mohammed-10ba5a197)
+ 
```

Comment:
Please use your correct information in the author section, link your LinkedIn profile, and link your GitHub profile.